tests/sentry/tasks/test_data_export.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import absolute_import
+
+from sentry.models import ExportedData, File
+from sentry.tasks.data_export import assemble_download, get_file_name
+from sentry.testutils import TestCase, SnubaTestCase
+
+
+class AssembleDownloadTest(TestCase, SnubaTestCase):
+    def setup(self):
```

Comment:
I think you want setUp(). That is the method unittest.TestCase uses. Remember to call super() as well.

Reply:
Gonna be honest, didn't even know that was a built-in method -- I was doing this because I though that's how I could separate things in tests

I'll refactor it!